src/apps/Settings/DaoSettings.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
 import styled from 'styled-components'
-import { toChecksumAddress } from 'web3-utils'
```

Comment:
Is the intention to only use `web3-utils` (the npm module) from `web3-utils.js`, so that we can move to another lib easily?

Reply:
Yes, and to avoid confusion :). There's too many `web3-utils` haha.